content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Outlook email notifications"
+linkTitle: "Configure outlook email notifications"
+weight: 40
+---
+
+## Configure outlook email notifications for pending pentest findings
+
+In order to configure Outlook to send emails every time a new finding gets created, we need to do the following:
+1. Create a Cobalt connection.
+2. Create an Outlook connection.
+3. Find and select a recipe (2 options):
+  A. [Cobalt > Outlook] When pentest finding moved to Pending Fix, send email notification
+  B. [Cobalt > Outlook] When DAST finding is detected, send email notification
+4. Copy and customize the recipe
+  1. Configure which pentest should trigger notifications when findings get created.
+  2. Configure which email address to send the to.
+5. Start the recipe
+
+The following is a step-by-step guide to walk you through the configuration process.
+
+### 1. Create a Cobalt connection.
+
+Find out how to create a Cobalt Connector [here](integrations/integrationbuilder/#connection).
+
+### 2. Create an Outlook connection.
+
+The following is a quick guide on how to create a Workato connection for Microsoft Outlook.
+In case you're looking for a more comprehensive and general guide on how to manage connections, check [here](integrations/integrationbuilder/connect-your-applications/).
+
+Go to the `Integration Builder` > `Projects`. Select or create a project on the left-side of the page. Then click on the "Create" button on the top-right and choose the "Connection" option.
+![Create a connection for your project](/integrations/integration_builder/findings_outlook_notifications/create_connection_step1.png "Create a connection for your project")
+
+A "New Connection" page will show up. Look for "Outlook" and select the corresponding tile. You can optionally use the search field to quickly find the connection.
+![Pick an Outlook connection](/integrations/integration_builder/findings_outlook_notifications/create_connection_step2.png "Pick an Outlook connection")
+
+After selecting the connection type, a form will appear. Provide a name, select a location and click on "Sign in with Windows". If needed, you can configure "Advanced settings" as well.
+![Configure the connection](/integrations/integration_builder/findings_outlook_notifications/create_connection_step3.png "Configure the connection")
```

Comment:
Please leave an extra empty line between the text and the image. Many images in this page appear inline and break the view.
```suggestion

{{% image src="/integrations/integration_builder/findings_outlook_notifications/create_connection_step3.png" alt="Configure the connection" %}}
```